validator/src/main.rs

```diff
@@ -1099,6 +1099,11 @@ pub fn main() {
                 .default_value("10000")
                 .help("Milliseconds between printing contact debug from gossip."),
         )
+        .arg(
+            Arg::with_name("ignore_poh_speed")
+                .long("ignore-poh-speed")
```

Comment:
How about something like this:
```suggestion
            Arg::with_name("disable_poh_calibration")
                .long("disable_poh_calibration")
```